src/lib.rs

```diff
@@ -166,18 +166,12 @@ pub fn formatter(
             e,
         ))
     })?;
-    let query = read_input(query).map_err(|e| {
-        FormatterError::Io(IoError::Filesystem(
-            "Failed to read query contents".into(),
```

Comment:
We may want to move this error message somewhere else now. I suppose now you don't get a very nice message if you reference a query file that doesn't exist.